media-sound/mpdscribble/mpdscribble-0.23.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson systemd
+
+DESCRIPTION="An MPD client that submits information to Audioscrobbler"
+HOMEPAGE="https://www.musicpd.org/clients/mpdscribble/"
+SRC_URI="https://www.musicpd.org/download/${PN}/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	media-libs/libmpdclient
+	net-misc/curl
+	dev-libs/boost
+	dev-libs/libgcrypt
+"
+
+DOCS=( AUTHORS COPYING NEWS README.rst )
+
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.23-Unconditionally-install-systemd-unit-files.patch
+	"${FILESDIR}"/${PN}-0.23-Don-t-install-AUTHORS-COPYING-NEWS-README.rst.patch
+)
+
+src_prepare() {
+	# we install these below via systemd_do{,user}unit
+	sed -i '/install_dir/d' -i systemd/*/meson.build
```

Comment:
Don't you already introduce some sort of patch file for this? Why's the sed needed anymore? Besides, external commands (`sed`) needs to `|| die`.